.rubocop.yml

```diff
@@ -3,20 +3,23 @@ require:
   - rubocop-performance
 
 AllCops:
-  TargetRubyVersion: 2.3.3
-
-Layout/LineLength:
-  Enabled: false
+  TargetRubyVersion: 2.5
 
 Layout/SpaceBeforeBlockBraces:
   Enabled: false
 
+Layout/LineLength:
+  Max: 120
+
 Metrics/MethodLength:
   Enabled: false
 
 Metrics/AbcSize:
   Enabled: false
 
+Naming/MethodParameterName:
+  MinNameLength: 1
```

Comment:
if it is for matrix indices, we can use `AllowedNames` parameter for allowing `i, j, k` but otherwise we think this cop is useful.